src/jsMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/internal/process.kt

```diff
@@ -0,0 +1,7 @@
+package it.krzeminski.snakeyaml.engine.kmp.internal
+
+external val process: Process
+
+external interface Process {
```

Comment:
question: is it possible to use `internal` modifier here? Right now they are public and, I believe, will be able for everyone (I don't think we want that)

Reply:
Sure, I moved them to system.js.kt, like we have with other `external` entities, just because I find it confusing to have `internal external` pair of modifiers 😆